src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -3082,7 +3082,7 @@ public function setRawAttributes(array $attributes, $sync = false)
      *
      * @param  string|null  $key
      * @param  mixed  $default
-     * @return array
+     * @return mixed|array
```

Comment:
`mixed|array` or `array|mixed` are used elsewhere in the codebase, and I think they make clearer there are different "signatures". Thoughts?